roles/cloud_connector/tasks/http_proxy.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Create systemd drop-in directory
+  ansible.builtin.file:
+    state: directory
+    path: /etc/systemd/system/rhcd.service.d
+    owner: root
+    group: root
+    mode: 0755
+
+- name: Deploy HTTP proxy systemd drop-in file
+  ansible.builtin.template:
+    src: "proxy.conf.j2"
+    dest: /etc/systemd/system/rhcd.service.d/proxy.conf
+    owner: root
+    group: root
+    mode: 0640
+
+- name: Restart and reload RHC service
+  ansible.builtin.systemd:
+    name: rhcd
+    state: restarted
```

Comment:
technically, it only needs restarting if the dropin changed, but oh well…

Reply:
Lets see if I handled handlers correctly

Reply:
It does read correct to me